docs/Contributing/API-for-contributors.md

```diff
@@ -3184,6 +3184,67 @@ Notifies the server about an agent error, resulting in two outcomes:
 
 `Status: 500`
 
+### Get the status of a device in the setup experience
+
+`GET /api/v1/fleet/device/:token/setup_experience/status`
```

Comment:
@marko-lisica just a heads-up, turns out this endpoint will be an orbit-authenticated one instead, and because those are always `POST` (due to the request payload containing the orbit key in JSON), this would look like `POST /api/fleet/orbit/setup_experience/status` instead.

Reply:
@mna Do we document Orbit-authenticated endpoints here or somewhere else?

Reply:
AFAIK we don't document them anywhere. Would be nice to keep it here for a bit though, just so I have the reference for the response payload :)

Reply:
@marko-lisica did this URL change?